packages/devtools_app/lib/src/shared/primitives/custom_pointer_scroll_view.dart

```diff
@@ -821,7 +821,7 @@ class _RenderScrollSemantics extends RenderProxyBox {
       if (child.isTagged(RenderViewport.excludeFromScrolling)) {
         excluded.add(child);
       } else {
-        if (!child.hasFlag(SemanticsFlag.isHidden)) {
+        if (!child.flagsCollection.isHidden) {
```

Comment:
`hasFlag` is deprecated in latest Flutter version, see https://github.com/flutter/flutter/issues/166101